test/resources/configurations/fetcher/FetcherConfTest8.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright 2016 LinkedIn Corp.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+
+<fetchers>
+    <fetcher>
+        <applicationtype>spark</applicationtype>
+        <classname>org.apache.spark.deploy.history.SparkFSFetcher</classname>
+        <params>
+            <namenode_addresses>sample-ha3.grid.company.com:50070,sample-ha4.grid.company.com:50070</namenode_addresses>
```

Comment:
This configuration should work with or without the IP failover model. In case of HA without IP failover, we can still check the addresses specified here and check for active namenode.